How-To-Fork-And-Maintain-a-Local-Instance-of-Free-Code-Camp.md

```diff
@@ -0,0 +1,70 @@
+# Overview
+If you plan to write a Pull Request for Free Code Camp, you will almost certainly need a local copy of the site. Having a local copy of the site will give you additional capability with Git that are not availible via the GitHub browser interface, including updating your fork and rebasing/squashing commits.
+
+This guide will cover how to fork the FCC project, clone a local copy, and how to maintain your fork. All Git commands will be given for the command line, which we strongly reccomend that you use, but all commands can be executed in a graphical Git enviornment as well.
+
+# Initial Setup
+### Setting Up your System
+1. Install [Git](https://git-scm.com/) or your favorite Git client
+1. (Optional) [Setup an SSH Key](https://help.github.com/articles/generating-ssh-keys/) for Github.  
+Using SSH can greatly speed up your interactions with GitHub, since you will not be prompted for your password.
+1. Create a parent projects directory on your system.  For the puposes of this document we will assume it is `/mean/`
+
+### Forking Free Code Camp
+1. Navigate to the top level Free Code Camp repository:  
+https://github.com/FreeCodeCamp/freecodecamp
+1. Click the "Fork" Button in the upper right hand corner of the interface
+1. After the project is forked, you will be taken to your copy of the FCC repo at `username/freecodecamp`
+
+### Cloning Your Fork
+1. From your fork of FCC, copy the HTTPS or SSH (if you installed SSH Keys) clone URL
```

Comment:
If it's not too much trouble it might be nice to have little screen snaps for these steps that ask a user to perform an action in the Github web UI.